<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTimerDemo</class>
 <widget class="QMainWindow" name="QTimerDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTimerDemo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>当前时间（时：分：秒）</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLCDNumber" name="hourLCD">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Mode::Dec</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::SegmentStyle::Filled</enum>
         </property>
         <property name="value" stdset="0">
          <double>23.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLCDNumber" name="secondLCD">
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="value" stdset="0">
          <double>23.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="minuteLCD">
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="value" stdset="0">
          <double>23.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>定时器</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="startBtn">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="singleBtn">
           <property name="text">
            <string>单次计时器</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>定时周期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="delaySpin">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <number>60000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="onceRadio">
           <property name="text">
            <string>单次计时</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="repeatRadio">
           <property name="text">
            <string>重复计时</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>定时器精度</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="preciseRadio">
         <property name="text">
          <string>精准</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="normalRadio">
         <property name="text">
          <string>一般</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="crudeRadio">
         <property name="text">
          <string>粗糙</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passtimeLabel">
      <property name="text">
       <string>流逝的时间：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
